src/pages/components/timeline.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Timeline
+description: >
+  The Timeline component lists events over a period of time.
+package: '@zendeskgarden/react-accordions'
+components:
+  - accordions/src/elements/timeline/Timeline.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To track an issue over time</li>
+      <li>To show events associated with an item</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To guide users through a step by step process, use a [Stepper](/components/stepper)
+          instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The timeline component is made up of individual timeline items. Item content appears to the right of the item marker by default.
+
+import TimelineDefault from '../../examples/accordions/timeline/TimelineDefault.tsx';
+import TimelineDefaultCode from '!!raw-loader!../../examples/accordions/timeline/TimelineDefault.tsx';
+
+<CodeExample code={TimelineDefaultCode}>
+  <TimelineDefault />
+</CodeExample>
+
+### Alternating
+
+Timeline items are displayed on alternating sides of the timeline.
+
+import TimelineAlternating from '../../examples/accordions/timeline/TimelineAlternating.tsx';
+import TimelineAlternatingCode from '!!raw-loader!../../examples/accordions/timeline/TimelineAlternating.tsx';
+
+<CodeExample code={TimelineAlternatingCode}>
+  <TimelineAlternating />
+</CodeExample>
+
+### Custom media
+
+import TimelineCustomMedia from '../../examples/accordions/timeline/TimelineCustomMedia.tsx';
+import TimelineCustomMediaCode from '!!raw-loader!../../examples/accordions/timeline/TimelineCustomMedia.tsx';
+
+<CodeExample code={TimelineCustomMediaCode}>
+  <TimelineCustomMedia />
+</CodeExample>
+
+### Opposite content
+
+In the opposite layout, a second content area is present on the opposite side of the timeline.
+
+import TimelineOppositeContent from '../../examples/accordions/timeline/TimelineOppositeContent.tsx';
+import TimelineOppositeContentCode from '!!raw-loader!../../examples/accordions/timeline/TimelineOppositeContent.tsx';
+
+<CodeExample code={TimelineOppositeContentCode}>
+  <TimelineOppositeContent />
+</CodeExample>
+
+## How to use it well
+
+### Nest content when necessary
+
+Components such as links, buttons, and images can be nested in each timeline item.
+
+<BestPractice>
+  <Do imageSource={props.data.timelineNestingDo.childFile.childImageSharp.gatsbyImageData}>
```

Comment:
It looks like there might be some issues with this image asset under ["Nest content when necessary"](https://6141ffefb486c356a65fb6fb--zendeskgarden.netlify.app/components/timeline#nest-content-when-necessary)? There are two vertical bars on the sides of the image, and the item timeline marker (circle) is not centered.

Reply:
@bethanybeech would have to help me with the centering of that marker in the figma asset :-)

Reply:
Both of these have been addressed, thanks @bethanybeech !